runtime/lua/vim/lsp/callbacks.lua

```diff
@@ -37,6 +37,19 @@ end
 
 M['textDocument/references'] = function(_, _, result)
   if not result then return end
+  for _, loc in ipairs(result) do
+    if loc.uri and not loc.message then
+      local bufnr = vim.uri_to_bufnr(loc.uri)
+      if not api.nvim_buf_is_loaded(bufnr) then
+        vim.fn.bufload(bufnr)
```

Comment:
I think if load unloaded buffers, should `bdelete` those buffers after get line.